crates/wysiwyg/src/dom/dom_handle.rs

```diff
@@ -90,6 +90,16 @@ impl DomHandle {
         DomHandle::from_raw(new_path.to_vec())
     }
 
+    /// Returns a DomHandle with the 'sub-path' down to the passed 'depth'.
+    /// i.e.: a DomHandle with path `[0, 1, 2, 3]` with `at_depth(2)` will return a DomHandle
+    /// with path `[3]`.
+    pub fn sub_handle_down_from(&self, depth: usize) -> DomHandle {
```

Comment:
I had been thinking about building something like this for the code blocks and quotes features. Thanks!